scrapers/MindGeek.yml

```diff
@@ -0,0 +1,132 @@
+      - milfhunter.com/scene/
+      - mofos.com/scene/
+      - mofosnetwork.com/scene/
+      - moneytalks.com/scene/
+      - publicagent.com/scene/
+      - publicpickups.com/scene/
+      - realityjunkies.com/scene/
+      - recklessinmiami.com/scene/
+      - rk.com/scene/
+      - roundandbrown.com/scene/
+      - seancody.com/scene/
+      - sneakysex.com/scene/
+      - sweetheartvideo.com/scene/
+      - sweetsinner.com/scene/
+      - teenslovehugecocks.com/scene/
+      - twistysnetwork.com/scene/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - 8thstreetlatinas.com/model/
+      - babesnetwork.com/model/
+      - bignaturals.com/model/
+      - captainstabbin.com/model/
+      - cumfiesta.com/model/
+      - devianthardcore.com/model/
+      - doghousedigital.com/
+      - dontbreakme.com/model/
+      - fakehostel.com/model/
+      - fakehub.com/model/
+      - faketaxi.com/model/
+      - familyhookups.com/model
+      - happytugs.com/model/
+      - iconmale.com/
+      - iknowthatgirl.com/model/
+      - letstryanal.com/model/
+      - lilhumpers.com/model/
+      - lookathernow.com/model/
+      - milfhunter.com/model/
+      - mofos.com/model/
+      - mofosnetwork.com/model/
+      - moneytalks.com/model/
+      - publicagent.com/model/
+      - publicpickups.com/model/
+      - recklessinmiami.com/model/
+      - rk.com/model/
+      - roundandbrown.com/model/
+      - seancody.com/model/
+      - sneakysex.com/model/
+      - sweetsinner.com/model/
+      - teenslovehugecocks.com/model/
+      - twistysnetwork.com/model/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $section: //img[@alt=""][contains(@src,"poster")]/../../.. #img>div>div>section
+    scene:
+      Title: $section//h1/text()|$section//h2/text()
+      Date:
+        selector: //div[text()="Release Date:"]/../text()
+        parseDate: January 2, 2006
+      Details: //div[text()="Description:"]/../div[not(@class)]/text()
+      Tags:
+        Name: //a[contains(@href,"/scenes?tags")]/text()[1]
+      Performers:
+        Name: $section//a[@href and @title]/text()
+      Studio:
+        Name: 
+            selector: $section//a[contains(@href,"site=")]/@title|//link[@rel="canonical"]/@href
+            postProcess:
+                - replace: 
+                    - regex: .*www\.
+                      with: ""
+                    - regex: \.com.+
+                      with: ""
+      Image: //img[@alt=""][contains(@src,"poster")]/@src
+
+  performerScraper:
+    performer:
+      Name: //section/div/h1/text()
+      Birthdate:
+        selector: //li[contains(.,"Date of Birth")]/span/text()[not(contains(.,"N/A"))]
+        postProcess:
+          - replace:
+              - regex: th,|st,|rd,|nd,
+                with: ","
+          - parseDate: January 2, 2006
+      Measurements: //li[contains(.,"Measurements")]/span/text()[not(contains(.,"N/A"))]
+      Country: 
```

Comment:
As all the sites here share the same data with brazzers, the country ( states for us born performers) could also use the mappings / postprocessing you already did in the brazzers performer scraper